src/main/java/org/broadinstitute/consent/http/models/DraftStudyDataset.java

```diff
@@ -0,0 +1,149 @@
+package org.broadinstitute.consent.http.models;
+
+import com.google.gson.Gson;
+import java.util.Date;
+import java.util.HashSet;
+import java.util.Optional;
+import java.util.Set;
+import java.util.UUID;
+import org.broadinstitute.consent.http.enumeration.DraftType;
+import org.broadinstitute.consent.http.models.dataset_registration_v1.DatasetRegistrationSchemaV1;
+
+/**
+ * Draft represents a partial submission of 0 or more elements. This is
+ * an internal structure, not intended to be serialized back to the client in the entirety.
+ */
+public class DraftStudyDataset implements DraftInterface {
```

Comment:
What does study dataset mean here? is it a draft study or a draft dataset or something else?